Revamp getting started section (#399)
subspace/subspace-docs: docs/learn/intro.md

- [Whitepaper - *Summarized*](https://subspace.network/news/subspace-network-whitepaper)
- [Whitepaper - *Full Length*](https://assets.website-files.com/61526a2af87a54e565b0ae92/617759c00edd0e3bd279aa29_Subspace_%20A%20solution%20to%20the%20farmer%27s%20dilemma.pdf)

## 🤝 Participate on the Subspace Network
---
### **Become a farmer**

> **Farmers** on the Subspace Network are responsible for maintaining consensus, or safety of the Consensus Chain. A **farmer** plots pieces of Archival History to disk, farms the created plot for block rewards, and joins the DSN as a node for data retrieval.

Reviewer: ```suggestion
**Farmers** on the Subspace Network are responsible for maintaining consensus, they do this by securing the **consensus chain**. A **farmer** plots pieces of **archival history** to disk, farms the created plot for block and vote rewards, and joins the DSN as a node for data retrieval.
```

Choose one of the following methods to **start farming** on the Network (from easy to more complex):

#### - [Start Farming with Pulsar](../farming-&-staking/farming/pulsar/pulsar-prerequisites)
#### - [Start Farming with Advanced CLI](../farming-&-staking/farming/advanced-cli/cli-install)

### **Become a nominator**

> Farmers can nominate **operators** and back them with their tokens, increasing their stake and chance of being elected as slot leaders. Generally speaking, any SSC token holder can stake their tokens by **nominating** a domain operator, without having to become an operator or farmer themselves. **Nominators** are earning **staking** rewards, propotional to their **staking amount** and **nominators tax**.

Reviewer: ```suggestion
Farmers can nominate **operators** and back them with their tokens, increasing their stake and chance of being elected as slot leaders. Generally speaking, any SSC token holder can stake their tokens by **nominating** a domain operator without having to become an operator or farmer themselves. **Nominators** earn **staking** rewards, proportional to their **staking amount** and pay the nominator a pre-agreed **tax** on them.
```

#### - [Start Nominating](../farming-&-staking/staking/)

### **Become an operator**

**Operators** are responsible for validating and executing transactions, producing execution receipts, applying state transitions and earn rewards for their work.

#### - [Start an Opeator node](../farming-&-staking/staking/operators/register-operator)

### **Become a timekeeper**

> **Timekeepers** on the Subspace Network responsible for running the **Proof-of-Time chain** and maintaining the randomness beacon for the **Consensus Chain**.

Reviewer: ```suggestion
**Timekeepers** on the Subspace Network are responsible for running the **Proof-of-Time chain** and maintaining the randomness beacon for the **consensus chain**.
```

#### - [Start timekeeping](../farming-&-staking/timekeeping)

> ## 📖 Develop on the Subspace Network

Reviewer: ```suggestion
## 📖 Develop on The Subspace Network
```

---

### Utilize the EVM Domain

> The Subspace Network has a running **EVM (Ethereum Virtual Machine)**, that allows you to deploy your **smart contracts.** 

Reviewer: ```suggestion
The Subspace Network hosts an **EVM (Ethereum Virtual Machine)**, that allows you to deploy your **smart contracts.** 
```

#### - [EVM Development](../develop/nova/intro)

### Utilize the Core Protocol

> The Subspace Network aims to provide an amazing developer experience to anyone who wishes to build on top of the protocol. As such we have started working on a variety of tools to help with development within our network. 

Reviewer: ```suggestion
The Subspace Network aims to provide an amazing developer experience to anyone who wishes to build on top of the protocol. As such we have started working on a variety of tools to assist and promote development on our network. 
```
